lvmd/device_class_manager.go

```diff
@@ -20,7 +26,16 @@ var qualifiedNameRegexp = regexp.MustCompile("^([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Z
 // This regexp is used to check StripeSize format
 var stripeSizeRegexp = regexp.MustCompile("(?i)^([0-9]*)(k|m|g|t|p|e|b|s)?$")
 
-// DeviceClass maps between device-classes and volume groups.
+// ThinPoolConfig holds the configuration of thin pool in a volume group
+type ThinPoolConfig struct {
+	// Name of thinpool
+	Name string `json:"name"`
+	// OverprovisionRatio signifies the upper bound multiplier for allowing logical volume creation in this pool
+	OverprovisionRatio *float64 `json:"overprovision-ratio"`
```

Comment:
```suggestion
	OverprovisionRatio float64 `json:"overprovision-ratio"`
```
This value looks not nullable.